packages/UsageAnalysis/files/Test Track/Viewers/Tree map viewer.md

```diff
@@ -0,0 +1,14 @@
+### Tree map viewer
+
+1. Open the **demog** dataset.
```

Comment:
Link dataset to the test case

Reply:
added   "datasets": ["System:DemoFiles/demog.csv"]